SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -70,29 +70,55 @@ class SPATIALGDK_API UEventTracingSamplingSettings : public UObject
 {
 	GENERATED_BODY()
 public:
+	struct TraceQueryDeleter
+	{
+		void operator()(Trace_Query* Query) const
+		{
+			if (Query != nullptr)
+			{
+				Trace_Query_Destroy(Query);
+			}
+		}
+	};
+
+	using TraceQueryPtr = TUniquePtr<Trace_Query, TraceQueryDeleter>;
+
 	UPROPERTY(EditAnywhere, Category = "Event Tracing", meta = (ClampMin = 0.0f, ClampMax = 1.0f))
 	double SamplingProbability = 1.0f;
 
 	UPROPERTY(EditAnywhere, Category = "Event Tracing")
 	TMap<FName, double> EventSamplingModeOverrides;
 
 	UPROPERTY(EditAnywhere, Category = "Event Tracing")
-	FString GDKEventPreFilter;
+	FString GDKEventPreFilter = "false";
```

Comment:
Minor: would prefer "None", a "false" filter seems ambiguous to me.

Reply:
Minor: can we make this a static string instead of c&p the same string?

Reply:
Will change to a static sting.

Won't be able to change to "None" as the filter string is passed into a worker API that handles it.

Reply:
Fair enough